.github/workflows/build-and-runtime-test.yml

```
@@ -42,11 +42,11 @@ jobs:
             build-tool: vuecli
     steps:
       - name: Checkout Amplify UI
-        uses: actions/checkout@v3
+        uses: actions/checkout@24cb9080177205b6e8c946b17badbe402adc938f #v3.4.0 https://github.com/actions/checkout/commit/24cb9080177205b6e8c946b17badbe402adc938f
```

One last nit, 
```suggestion
        uses: actions/checkout@24cb9080177205b6e8c946b17badbe402adc938f # v3.4.0 https://github.com/actions/checkout/commit/24cb9080177205b6e8c946b17badbe402adc938f
```
and same with others

---

Fixed spacing